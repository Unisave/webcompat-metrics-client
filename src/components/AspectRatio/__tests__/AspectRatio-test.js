/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */

import React from "react";
import renderer from "react-test-renderer";

import AspectRatio from "..";

it("renders AspectRatio default correctly ", () => {
  const tree = renderer
    .create(
      <AspectRatio>
        <div>{"test"}</div>
      </AspectRatio>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders AspectRatio width 50% ratio correctly ", () => {
  const tree = renderer
    .create(
      <AspectRatio ratio={"50%"}>
        <div>{"test"}</div>
      </AspectRatio>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
