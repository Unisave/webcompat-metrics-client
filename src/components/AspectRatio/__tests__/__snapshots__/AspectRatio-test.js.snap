// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders AspectRatio default correctly  1`] = `
<div
  className="component"
>
  <div
    className="ratio"
    style={
      Object {
        "paddingBottom": "100%",
      }
    }
  />
  <div
    className="content"
  >
    <div>
      test
    </div>
  </div>
</div>
`;

exports[`renders AspectRatio width 50% ratio correctly  1`] = `
<div
  className="component"
>
  <div
    className="ratio"
    style={
      Object {
        "paddingBottom": "50%",
      }
    }
  />
  <div
    className="content"
  >
    <div>
      test
    </div>
  </div>
</div>
`;
